### PR TITLE
Content

### PR DESCRIPTION
**Pull request by leonardofrp5**
**Summary**

Design the content of technologies

**Trello Card**

https://trello.com/c/APlNtmmE/23-content-layout

**Screenshots**

captura de pantalla 2018-11-01 a la s 11 15 48

### CODE REVIEW

File: maquetado/index.html [hunk 1]
```diff
@@ -9,6 +9,85 @@
...
+        <p class="tech-content">
+          Polymer is an open-source JavaScript library for building web applications using Web Components.
+        </p>
+      </div>   
+    </section>
+    <h1 class="title">Back-End Tools</h1>
+    <section class="section-tech">
+      <div class="tech">
+        <img class="tech-logo" alt="Rails logo" src="assets/rails_icon.png">
+        <h4 class="tech-title">Polymer</h4>
```
Comment (henryzarza) [inline maquetado/index.html]: Change this for `Ruby on Rails`

File: maquetado/index.html [hunk 2]
```diff
@@ -9,6 +9,85 @@
...
+        <h5 class="tech-title title-bold">Release: 2015</h5>
+        <p class="tech-content">
+          Polymer is an open-source JavaScript library for building web applications using Web Components.
```
Comment (henryzarza) [inline maquetado/index.html]: Change this for the description of `Rails`

File: maquetado/index.html [hunk 3]
```diff
@@ -9,6 +9,85 @@
...
+        </p>
+      </div>
+      <div class="tech">
+        <img class="tech-logo" alt="Express logo" src="assets/express_icon.svg">
+        <h4 class="tech-title">Polymer</h4>
```
Comment (henryzarza) [inline maquetado/index.html]: The same case here but with `Express`

File: maquetado/index.html [hunk 4]
```diff
@@ -9,6 +9,85 @@
...
+        <h5 class="tech-title title-bold">Release: 2015</h5>
+        <p class="tech-content">
+          Polymer is an open-source JavaScript library for building web applications using Web Components.
```
Comment (henryzarza) [inline maquetado/index.html]: And here

File: maquetado/index.html [hunk 5]
```diff
@@ -9,6 +9,85 @@
...
+        </p>
+      </div>
+      <div class="tech">
+        <img class="tech-logo" alt="Aspnet logo" src="assets/aspnet_icon.png">
+        <h4 class="tech-title">Polymer</h4>
```
Comment (henryzarza) [inline maquetado/index.html]: The same here but with `ASP.NET`

File: maquetado/index.html [hunk 6]
```diff
@@ -9,6 +9,85 @@
...
+        <h5 class="tech-title title-bold">Release: 2015</h5>
+        <p class="tech-content">
+          Polymer is an open-source JavaScript library for building web applications using Web Components.
```
Comment (henryzarza) [inline maquetado/index.html]: And here

File: maquetado/index.html [hunk 7]
```diff
@@ -9,6 +9,85 @@
...
+        </p>
+      </div>
+      <div class="tech">
+        <img class="tech-logo" alt="Django logo" src="assets/django_icon.svg">
+        <h4 class="tech-title">Polymer</h4>
```
Comment (henryzarza) [inline maquetado/index.html]: And here but with `Django`

File: maquetado/index.html [hunk 8]
```diff
@@ -9,6 +9,85 @@
...
+        <h5 class="tech-title title-bold">Release: 2015</h5>
+        <p class="tech-content">
+          Polymer is an open-source JavaScript library for building web applications using Web Components.
```
Comment (henryzarza) [inline maquetado/index.html]: And here

File: maquetado/scss/_tech.scss [hunk 9]
```diff
@@ -0,0 +1,68 @@
+.content {
+  margin: 30px auto;
+  max-width: 100%;
+  width: 1024px;
+}
+.title {
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss:7]: Add a break line here

File: maquetado/scss/_tech.scss [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+  width: 100%;
+}
+.section-tech {
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss:11]: Add break line here

File: maquetado/scss/_tech.scss [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+  display: flex;
+  flex-direction: row;
+  flex-flow: wrap;
+  justify-content: space-between;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss:15]: I think that this isn't necessary, delete it

File: maquetado/scss/_tech.scss [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+  padding: 40px;
+  width: 100%;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: The same here

File: maquetado/scss/_tech.scss [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+}
+.title {
+  width: 100%;
+}
+.section-tech {
+  display: flex;
+  flex-direction: row;
+  flex-flow: wrap;
+  justify-content: space-between;
+  padding: 40px;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: Change this property for `margin` property

File: maquetado/scss/_tech.scss [hunk 14]
```diff
@@ -0,0 +1,68 @@
...
+  &::before {
+    border: solid 1px #BED23C;
+    border-radius: 5px;
+    bottom: 5px;
+    content: '';
+    height: 100%;
+    position: absolute;
+    right: 7px;
+    width: 100%;
+  };
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: This `;` isn't necessary delete it

File: maquetado/scss/_tech.scss [hunk 15]
```diff
@@ -0,0 +1,68 @@
...
+.tech-content {
+  font-size: $text-small;
+  padding: 10px 19px;
+}
+
+.tech-logo {
+  height: $tech-logo-size;
+  width: $tech-logo-size;
+}
+.express-sty,
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: I think that isn't necessary, please delete it

**Review by henryzarza — CHANGES_REQUESTED** [review]
Great :+1:

File: maquetado/scss/_tech.scss (hunk 11 above)
Comment (leonardofrp5) [reply maquetado/scss/_tech.scss:15]: I didn't delete this line because is necessary for the space between cards of tecnologia

**Pull request merged**